juicy-ace-editor.html

```diff
@@ -47,7 +47,9 @@
         }
         set value(val){
             if(this.editor){
+                this.ignoreChange = true;
```

Comment:
I know Tomek would suggest adding an underscore to make sure it's not part of the API

```suggestion
                this._ignoreChange = true;
```